certified-connectors/RegexFlow Regular Expression/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key (https://portal.publicapi.regexflow.com/)",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#6C2C74",
+    "capabilities": [],
+    "publisher": "Imre Agocs"
```

Comment:
@imreagocs-regexflow publisher property should contain company name since this is a certified connector.
And stackOwner property is missing here. Please refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json) for more information about these two properties.

Reply:
fixed